tests/common.go

```diff
@@ -227,23 +254,29 @@ func DeleteVolumesAndWait(ctx *scheduler.Context) {
 }
 
 // ScheduleAndValidate schedules and validates applications
-func ScheduleAndValidate(testname string) []*scheduler.Context {
+func ScheduleAndValidate(testname string,vps_map map[string]string,vState int) []*scheduler.Context {
```

Comment:
do not change ScheduleAndValidate common function just for your test, use **Inst().S.Schedule(taskName, scheduler.ScheduleOptions{...}** from your test directly with **ValidateContext(ctx,vState)**. In this case you don't need to modify all ScheduleAndValidate invocation in each tests.